lib/cisco_node_utils/bgp_af.rb

```diff
@@ -500,8 +491,7 @@ def default_maximum_paths_ibgp
 
     # Build an array of all network commands currently on the device
     def networks
-      cmds = config_get('bgp_af', 'network', @get_args)
-      cmds.nil? ? default_networks : cmds.each(&:compact!)
+      config_get('bgp_af', 'network', @get_args).each(&:compact!)
```

Comment:
[Per the Ruby documentation](http://ruby-doc.org/core-2.2.0/Array.html#method-i-compact-21), the compact method `Returns nil if no changes were made, otherwise returns the array.`

To avoid returning nil:

```
      cmds = config_get('bgp_af', 'network', @get_args)
-     cmds.nil? ? default_networks : cmds.each(&:compact!)
+     cmds.empty? ? cmds : cmds.each(&:compact!)
```